tests/tests_twobody/test_orbit.py

```diff
@@ -173,7 +173,9 @@ def test_apply_maneuver_changes_epoch():
 def test_orbit_from_ephem_with_no_epoch_is_today():
     # This is not that obvious http://stackoverflow.com/q/6407362/554319
     body = Earth
-    ss = Orbit.from_body_ephem(body)
+    epoch = Time.now().tdb
+    ephem = Ephem.from_body(body, epoch)
+    ss = Orbit.from_ephem(Sun, ephem, epoch)
```

Comment:
The tests of `Orbit.from_body_ephem` require further discussion.

If we are still using it internally, even if it's deprecated, it should be tested. And if it has to be tested, then we should silence the deprecation warnings.

But if we are not using it internally anymore, then this approach is on the right path, but I would try not to call `Ephem.from_body` (and use a custom, dummy `Ephem`).